src/observable/dom/AjaxObservable.ts

```diff
@@ -417,7 +417,7 @@ export class AjaxResponse {
           this.response = JSON.parse(xhr.responseText || 'null');
         }
         break;
-      case 'xml':
+      case 'document':
```

Comment:
Okay... can we leave `'xml'` in there for now, with a // TODO: to remove it? That will keep compatability and make this a patch rather than a major revision change. We can remove `xml` in the next major version.